packages/core/src/statistics/lidoStatistics.ts

```diff
@@ -12,7 +12,7 @@ export async function getLidoStatistics(this: SolidoSDK) {
   ]);
 
   return {
-    apy,
+    stakeApy: stakeApy.max,
```

Comment:
Why `stakeApy`? I understand that it would be a major versions
But is this change really necessary?